src/DotVVM.Framework/Resources/Scripts/DotVVM.ts

```diff
@@ -386,12 +386,23 @@ class DotVVM {
         return vm.$csrfToken
     }
 
-    public staticCommandPostback(viewModelName: string, sender: HTMLElement, command: string, args: any[], callback = _ => { }, errorCallback = (errorInfo: {xhr: XMLHttpRequest, error?: any}) => { }) {
+    public staticCommandPostback(viewModelName: string, sender: HTMLElement, command: string, args: any[], callback = _ => { }, errorCallback = (errorInfo: {xhr?: XMLHttpRequest, error?: any}) => { }) {
         (async () => {
+            var csrfToken;
+            try {
+                csrfToken = await this.fetchCsrfToken(viewModelName);
+            }
+            catch (err) {
+                this.events.error.trigger(new DotvvmErrorEventArgs(sender, this.viewModels[viewModelName].viewModel, viewModelName, null, null));
+                console.warn(`CSRF token fetch failed.`);
+                errorCallback({ error: err });
+                return;
+            }
```

Comment:
Might have been cleaner to use `await this.fetchCsrfToken(viewModelName).catch(err => ...)`

Reply:
I didn't knew about that, but still - I need to use `return` in the catch clause so it doesn't help much here.